mapusaurus/respondents/views.py

```diff
@@ -165,6 +165,18 @@ def search_results(request):
          'total_pages': total_pages, 'current_sort': current_sort},
         template_name='respondents/search_results.html')
 
+def get_lender_hierarchy(inst, exclude, order_by):
+    lender_hierarchy = inst.lenderhierarchy_set.first()
+    if lender_hierarchy:
+        org_id = lender_hierarchy.organization_id
+        hierarchy_list = LenderHierarchy.objects.filter(organization_id=org_id).select_related('institution')
+        if exclude:
+            hierarchy_list = hierarchy_list.exclude(institution=inst)
+        if order_by:
+            hierarchy_list = hierarchy_list.order_by('-institution__assets')
+        return hierarchy_list
+    return [] 
+
```

Comment:
It seems like this might be a better fit as a function inside the Institution class?